corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html

```diff
@@ -1,6 +1,6 @@
 {% load hq_shared_tags %}
 
-<div class="charts-pie-chart"
+<div class="charts-multibar-chart"
```

Comment:
lol is this why that report wasn't showing anything?

Reply:
yep :D

Reply:
😂